src/test/java/com/wikia/webdriver/common/contentpatterns/AdSlot.java

```diff
@@ -0,0 +1,29 @@
+package com.wikia.webdriver.common.contentpatterns;
+
+import lombok.Getter;
+
+public enum AdSlot {
+  TOP_LEADERBOARD("top_leaderboard","div[id$='top_leaderboard_0__container__']"),
+  BOTTOM_LEADERBOARD("bottom_leaderboard","div[id*='wka1a.PF/bottom_leaderboard'][id*='_0__container__']"),
+  TOP_BOXAD("top_boxad", "div[id$='top_boxad']"),
+  INCONTENT_BOXAD("incontent_boxad", "div[id$='incontent_boxad']"),
+  BOTTOM_BOXAD("bottom_boxad", "div[id$='bottom_boxad_0__container__']"),
+  FEED_BOXAD("feed_boxad", "div[id$='google_ads_iframe_/5441/wka1a.PF/feed_boxad/desktop/ns-article/_fandom-all_0__container__']"),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "feed_boxad" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)